backend/src/main/kotlin/pl/edu/agh/backend/search/MicroserviceCommunicationService.kt

```diff
@@ -0,0 +1,26 @@
+package pl.edu.agh.backend.search
+
+import org.springframework.boot.web.client.RestTemplateBuilder
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestTemplate
+import org.springframework.web.client.getForObject
+
+@Service
+class MicroserviceCommunicationService {
+
+    val servicesNames : List<String> = listOf("text-search:80")
+
+    val restTemplate: RestTemplate = RestTemplateBuilder().build()
+
+    fun getResponse(phrase : String, rootPath : String): List<String?> {
+        val results = mutableListOf<String>()
+
+        servicesNames.forEach {
```

Comment:
- jeśli chcemy na wyjściu tablicę, chyba ładniej by było użyć poprostu map?
- średnio podoba mi się, że w tej tablicy w JSON są stringi, a nie obiekty, ale chyba może to jak na razie zostać